fontforge/sfd.c

```diff
@@ -3982,8 +3984,8 @@ static SplineSet *SFDGetSplineSet(FILE *sfd,int order2) {
 		    SplinePointList *spl = chunkalloc(sizeof(SplinePointList));
 		    spl->first = spl->last = pt;
 		    spl->start_offset = 0;
-		    if ( cur!=NULL ) {
-			if ( SFDCloseCheck(cur,order2))
+		    if ( cur!=NULL && !(have_cur_val==1 && cur_val & SFD_PTFLAG_FORCE_OPEN_PATH) ) {
+			if ( SFDCloseCheck(cur,order2) )
 			    --ttfindex;
```

Comment:
If there is a "move" operation, we need to reset the path state since we are on a new path now, right?

Reply:
@andreygursky, ping.

Reply:
@andreygursky, ping.